x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/types.ts

```diff
@@ -133,6 +133,11 @@ export interface AboutStepRiskScore {
   isMappingChecked: boolean;
 }
 
+export enum DataSourceType {
+  IndexPatterns = 'indexPatterns',
+  DataView = 'dataView',
+}
```

Comment:
Is this defined on the FE side only, or at the API and/or BE level as well?

We might benefit from defining it under the `common` folder because we need to distinguish rules with these two types of data sources in tests as well, for example: https://github.com/elastic/kibana/pull/136822#discussion_r927875890